components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -1359,4 +1427,41 @@ private String getUsername(DocumentReference reference)
         }
         return reference.getName();
     }
+
+    private Set<String> addHeaders(String[] fieldIds, String[][] headerIds, Set<String> enabledFields)
+    {
+        Set<String> present = new HashSet<String>();
+        int counter = 0;
+        for (String fieldId : fieldIds) {
+            if (enabledFields.remove(fieldId)) {
+                for (String headerId : headerIds[counter]) {
+                    present.add(headerId);
+                }
+            }
+            counter++;
+        }
+        return present;
+    }
+
+    private String parseEvidence(String value)
+    {
+        Map<String, String> valueTranslates = new HashMap<String, String>();
+        valueTranslates.put("rare", "Rare (MAF<0.01); ");
+        valueTranslates.put("predicted", "Predicted damaging by in silico models; ");
+        valueTranslates.put("reported", "Reported in other individuals; ");
+        valueTranslates.put("segregates", "Segregates with phenotype within family; ");
```

Comment:
Should I make a separate issue to translate all places in a file? Seems nothing is translated there yet (see phenotypeHeader(), documentInfoHeader(),patientInfoHeader(), familyHistoryHeader(), etc).